k8s/pkg/apis/longhorn/v1beta2/node.go

```diff
@@ -6,6 +6,7 @@ const (
 	NodeConditionTypeReady            = "Ready"
 	NodeConditionTypeMountPropagation = "MountPropagation"
 	NodeConditionTypeSchedulable      = "Schedulable"
+	NodeConditionTypeEnvironmentCheck = "EnvironmentCheck"
```

Comment:
Should we use the name `EnvironmentCheckPassed` instead of `EnvironmentCheck` so that the condition will read like this:
```
    - lastProbeTime: ""
      lastTransitionTime: "2024-02-23T20:44:36Z"
      message: Node jbm-u22-pool2-ca5a5aa4-5fcq7 running kernel 5.15.0-67-generic
      reason: ""
      status: "True"
      type: EnvironmentCheckPassed
```

Reply:
Done.